backend/battle/battles/battle.py

```diff
@@ -12,13 +12,19 @@ def run_battle(battle):
 
 
 def get_winner_for(team_creator, team_opponent):
-    creator_pokemons = team_creator.pokemons.all()
-    opponent_pokemons = team_opponent.pokemons.all()
+
+    pkn_team_cre = team_creator.teams.all().order_by('order').prefetch_related('pokemon')
```

Comment:
Those abbreviations are a bit unreadable IMO. It's better to use names like `creator_pkn_team_qs` for example.

Reply:
I put the abbreviations because pylint was giving many characters error, so I need to shorten the variable names as much as possible

Reply:
That's a bad pylint error. You can reconfigure it to avoid it. LMK if you have trouble.

Reply:
Yes, this used to complicate my development, but I remember Xima commented this limit of character is a good practice, should I reconfigure the limit to more or change my variable names?

Reply:
cool, configure to more, like 25

Reply:
thanks :)